pkg/lib/oauth/oidc/id_token.go

```diff
@@ -48,11 +94,16 @@ func (ti *IDTokenIssuer) IssueIDToken(client *config.OAuthClientConfig, s sessio
 	now := ti.Clock.NowUTC()
 
 	_ = claims.Set(jwt.AudienceKey, client.ClientID)
+
+	_ = claims.Set("sid", EncodeSID(s))
```

Comment:
Seems `sid` is custom claim. Will we don't some prefix to it, do we have similar case in the existing code base?

https://openid.net/specs/openid-connect-core-1_0.html#IDToken

Reply:
No. It is defined here. https://openid.net/specs/openid-connect-frontchannel-1_0.html#OPLogout